vector/src/main/java/im/vector/app/features/call/CallAudioManager.kt

```diff
@@ -246,7 +246,7 @@ class CallAudioManager(
     }
 
     private fun isHeadsetOn(): Boolean {
-        return isWiredHeadsetOn() || isBluetoothHeadsetOn()
+        return isWiredHeadsetOn() || (audioManager?.let { isBluetoothHeadsetConnected(it) } ?: false)
```

Comment:
`audioManager` should only be used from a single thread, according to Valere's comment.
The fix LGTM, but let discuss with Valere about it.

Reply:
Should be ok to read state

Reply:
Thanks